.github/workflows/kiali-ci.yml

```
@@ -202,5 +202,5 @@ jobs:
         working-directory: tests/integration/tests
 
       - name: Get kiali pod logs when tests fail
-        if: ${{ steps.integration-tests.outcome == 'failure' }}
+        if: steps.integration-tests.outcome == 'failure'
         run: kubectl logs deployments/kiali -n istio-system
```

this is weird, i thought we already changed this? Yesterday @leandroberetta had changed this, I thought? This is to ensure we get the logs regardless of the `deployment.instance_name` we may have tested with:
```suggestion
        run: kubectl logs -l app.kubernetes.io/name=kiali -n istio-system
```

---

@jmazzitelli my PR is not merged yet, changes are still there.